kernel/kernel-applicanttype-api/src/main/java/io/mosip/kernel/applicanttype/api/impl/ApplicantTypeImpl.java

```diff
@@ -26,189 +32,68 @@
-			}
-		} catch (Exception e) {
-			LOGGER.error("Error while claculating age");
-			throw new InvalidApplicantArgumentException(
-					ApplicantTypeErrorCode.INVALID_DATE_STRING_EXCEPTION.getErrorCode(),
-					ApplicantTypeErrorCode.INVALID_DATE_STRING_EXCEPTION.getErrorMessage(), e);
-		}
-
-		String ageCode = null;
+			BufferedReader br = new BufferedReader(new FileReader(p.toFile()));
 
-		try {
-			if (age >= Integer.parseInt(ageLimit)) {
-				ageCode = ADULT;
-			}
-			if (age >= 0 && age < Integer.parseInt(ageLimit)) {
-				ageCode = CHILD;
+			String line = br.readLine();
+			while (null != line) {
+				sb.append(line);
+				line = br.readLine();
 			}
-		} catch (NumberFormatException e) {
-			LOGGER.error("Error while setting age code");
-			throw new InvalidApplicantArgumentException(ApplicantTypeErrorCode.INVALID_QUERY_EXCEPTION.getErrorCode(),
-					ApplicantTypeErrorCode.INVALID_QUERY_EXCEPTION.getErrorMessage(), e);
-		}
 
-		// check and return the applicant id
-		return findApplicantType(itc, genderType, isBioExPresent, ageCode);
-	}
-
-	private boolean isNullEmpty(String str) {
-		return str == null || str.trim().length() <= 0;
-	}
-
-	private String findApplicantType(String itc, String genderType, boolean isBioExPresent, String ageCode) {
-		if (itc.equals(FOREIGNER) && genderType.equals(MALE) && ageCode.equals(CHILD) && !isBioExPresent) {
-			// 1
-			return "001";
-		} else if (itc.equals(FOREIGNER) && genderType.equals(MALE) && ageCode.equals(ADULT) && !isBioExPresent) {
-			// 2
-			return "002";
-
-		} else if (itc.equals(NON_FOREIGNER) && genderType.equals(MALE) && ageCode.equals(CHILD) && !isBioExPresent) {
-			// 3
-			return "003";
-
-		} else if (itc.equals(NON_FOREIGNER) && genderType.equals(MALE) && ageCode.equals(ADULT) && !isBioExPresent) {
-			// 4
-			return "004";
-
-		} else if (itc.equals(FOREIGNER) && genderType.equals(FEMALE) && ageCode.equals(CHILD) && !isBioExPresent) {
-			// 5
-			return "005";
-
-		} else if (itc.equals(FOREIGNER) && genderType.equals(FEMALE) && ageCode.equals(ADULT) && !isBioExPresent) {
-			// 6
-			return "006";
-
-		} else if (itc.equals(NON_FOREIGNER) && genderType.equals(FEMALE) && ageCode.equals(CHILD) && !isBioExPresent) {
-			// 7
-			return "007";
-
-		} else if (itc.equals(NON_FOREIGNER) && genderType.equals(FEMALE) && ageCode.equals(ADULT) && !isBioExPresent) {
-			// 8
-			return "008";
-
-		} else if (itc.equals(FOREIGNER) && genderType.equals(MALE) && ageCode.equals(CHILD) && isBioExPresent) {
-			// 9
-			return "009";
-
-		} else if (itc.equals(FOREIGNER) && genderType.equals(MALE) && ageCode.equals(ADULT) && isBioExPresent) {
-			// 10
-			return "010";
-
-		} else if (itc.equals(NON_FOREIGNER) && genderType.equals(MALE) && ageCode.equals(CHILD) && isBioExPresent) {
-			// 11
-			return "011";
+		} catch (Exception e) {
+			e.printStackTrace();
+		}*/
+		String mvelExpression = restTemplate.getForObject(configServerFileStorageURL + mvelFile, String.class);

```

Comment:
@Shilpa-Manjunath Can we add exeption handling in case of client or server exeption and add logs.